src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -26,6 +38,18 @@ Describe "Arcus" {
                 Assert-MockCalled Get-AzLogicAppRunHistory -Times 0
                 Assert-MockCalled Get-AzLogicApp -Times 0
             }
+            It "Doesn't enable anything when both checkType & stopType is 'None'" {
+                # Arrange
+                Mock Get-AzLogicAppRunHistory {}
```

Comment:
Some backdoor parameter assertion can be added here so we can verify that the correct values are send to these mocked functions. (Like `State = Enabled`, `ResourceGrouop = 'ignored-resource-group'`)

Reply:
Good remark, let's come back to this and perhaps review all tests to include these type of assertions from a different issue?

Reply:
Hmm, as far as I know we've already added these backdoor assertions in all tests. But ok.